mixin/build.go

```diff
@@ -161,7 +161,7 @@ func buildMix(prepNeeded bool) error {
 		_ = os.Remove(mixFlagFile)
 		return err
 	}
-	err = b.BuildUpdate("", 0, "", false, true, false)
+	err = b.BuildUpdate("", 0, "", false, true, false, false, false)
```

Comment:
Oh man, this function signature is just ridiculous now ❌ ahah. Can we please figure out a better way to call this, or change every single parameter to a named one so it's not insane to call in a future spot? FTFFF means nothing to even me at this point 😅

Reply:
Maybe a struct containing the configuration for this specific command that contains all the bools necessary?

Reply:
it is! I wasn't sure if I wanted to add to the mess or just clean it up at once. I took the easy path. ;)

Probably a better approach would be to set those properties in the builder object.

Reply:
I don't blame you, but I like the rest of the code so if you can make it pretty in this commit that'd be awesomeeeeee!

Reply:
I just noticed that two of those parameters (prefixflag and keepChroots)  are not even used anymore....

Reply:
😆 😂

Reply:
https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis and the associated video is worth reviewing if you want to do better than just adding a struct, although if I was doing it I would just add a struct.